Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -48,12 +48,6 @@ If this value is empty, each pod will get an ephemeral directory to store their
     * `enabled`: Whether to enable prometheus based monitoring for external or internal cluster
     * `externalMgrEndpoints`: external cluster manager endpoints
     * `externalMgrPrometheusPort`: external prometheus manager module port. See [external cluster configuration](#external-cluster) for more details.
-    * `rulesNamespace`: Namespace to deploy prometheusRule. If empty, namespace of the cluster will be used.
-      Recommended:
-        * If you have a single Rook cluster, set the `rulesNamespace` to the same namespace as the cluster or keep it empty.
-        * If you have multiple Rook clusters in the same Kubernetes cluster, choose the same namespace to set `rulesNamespace` for all the clusters (ideally, namespace with prometheus deployed). Otherwise, you will get duplicate alerts with duplicate alert definitions.
-    * `port`: The internal prometheus manager module port where the prometheus mgr module listens. The port may need to be configured when host networking is enabled.
```

Comment:
`port` and `interval` are still valid in the cluster CR for monitoring, were these two lines accidentally deleted?

Reply:
Yaa right,
Updated